wiki/Hit_Objects/Slider_Tick_Rate/en.md

```diff
@@ -0,0 +1,11 @@
+---
+tags:
+- ticks
+---
+
+# Slider Tick Rate
```

Comment:
probably belongs in /wiki/Beatmapping with the other map/timing settings. you have enough info about ticks in Hit_Objects/Slider already IMO